app/src/org/commcare/tasks/ResourceEngineTask.java

```diff
@@ -129,7 +126,7 @@ public void compoundResourceAdded(final ResourceTable table) {
         synchronized (statusLock) {
             // if last time isn't set or is less than our spacing count, do not
             // perform status update. Also if we are already running one, just skip this.
-            if (statusCheckRunning || System.currentTimeMillis() - lastTime < ResourceEngineTask.STATUS_UPDATE_WAIT_TIME) {

```

Comment:
Clayton added this threaded process so that updating the install progress wouldn't block actually performing the install. He limited such that it only ran once every second. Since I've updated entire recalculations to only happen when you add a 'compound' resources (media suite, suite, and profile resources), we shouldn't need a timeout prevent how often this runs because for most apps it should only run 2-3 times an install.